epsagon/runners/tornado.py

```diff
@@ -83,10 +96,16 @@ def update_response(self, response, response_body=None):
         )
 
         if response_body:
+            body = response_body
+            if isinstance(body, list):
+                body = body[0]
```

Comment:
in which cases is the body list? (wondering whether we want to collect all the list items)

Reply:
It seems to be always a list from what I've seen, and always a single element